add a note about log tags for datadog task logs (#2905)
astronomer/docs: astro/view-logs.md

  :::

3. (Optional) Set the following [environment variable](environment-variables.md) on your Deployment to send your logs to a specific [Datadog site](https://docs.datadoghq.com/getting_started/site/):

    - **Key**: `DATADOG_SITE`
    - **Value**: Your Datadog site name. For example, `US3`.
   
4. (Optional) Set the following [environment variable](environment-variables.md) on your Deployment to [add specific tags to your logs](https://docs.datadoghq.com/getting_started/tagging/):

    - **Key**: `ASTRO_DATADOG_TASK_LOGS_TAGS`
    - **Value**: `<tag-key-1>:<tag-value-1>,<tag-key-2>:<tag-value-2>`

> By default, the tags `source=astronomer` and `service=astronomer-task-logs` will be used.

Reviewer: ```suggestion
By default, Astro uses the tags `source=astronomer` and `service=astronomer-task-logs`.
```

Author: Astro does. good suggestion
